content/en/docs/concepts/architecture/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,79 @@
+---
+reviewers:
+- jpbetz
+title: Unknown Version Interoperability Proxy
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that lets one
+{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+proxy a resource request to a different _peer_ API server. This is useful when there are multiple
+API servers running different versions of Kubernetes in one cluster (for example, during a long-lived
+rollout to a new release of Kubernetes).
+
+This mechanism is called the _Unknown Version Interoperability Proxy_.
+## Enabling the Unknown Version Interoperability Proxy
+The Unknown Version Interoperability Proxy feature is controlled by a
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+and is not enabled by default. You can activate Unknown Version Interoperability Proxy by enabling
+the feature gate named `UnknownVersionInteroperabilityProxy` when you start the
+{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}:
+
+```shell
+kube-apiserver \
+--feature-gates=UnknownVersionInteroperabilityProxy=true \
+# required flag for this feature
+--peer-ca-file=`ca bundle to verify serving certs of kube-apiserver`
+# optional flags for this feature
+--peer-advertise-ip=`ip of this kube-apiserver that should be used by peers to proxy requests`
+--peer-advertise-port=`port of this kube-apiserver that should be used by peers to proxy requests`
+# …and other flags as usual
+```
+
+## Interoperability proxying
```

Comment:
A picture would be worth 1000 words for sure here. Can we add a simplified diagram that highlights how a proxied request works (from a cluster administrator's perspective, where the apiserver binary is a black box)?

Reply:
![image](https://github.com/kubernetes/website/assets/26771552/07857533-bf53-4a54-8545-cd8eb8c8b41a)

Reply:
How does this look?

Reply:
(not needed for alpha, great to have though)

Reply:
For tips, see https://www.k8s.dev/blog/2021/12/01/improve-your-documentation-with-mermaid.js-diagrams/

Reply:
![image](https://github.com/kubernetes/website/assets/26771552/67802727-ac6b-4863-8d66-56203e92adde)

tried my hand at mermaid